### DISCUSSION TITLE
Steal-puffle: nueva posición de puffle robado

### DISCUSSION THREAD

**Discussion by amanda1415** [IIC2133-PUC/2024-2#57] [Tarea 0] [answered]
Hola! Me queda una duda respecto a este evento. 

Cuando el pingüino 1 le roba puffles al pinguino 2, estos deben quedar en la misma posición que usaban anteriormente en la lista del pingüino 2 (como el caso de el evento TRADE-PUFFLE), o es de libre elección? 

Tengo la misma duda para el evento de GIVE-PUFFLE.

Quedo atenta, gracias!

**Comment by tomastrivino** [comment, marked as answer]
Hola,
véase #16 , #23 
Cuéntanos si mantienes la duda
Saludos!
